packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -726,7 +723,7 @@ export class ConnectionManager implements IConnectionManager {
 
         this.disconnectFromDeltaStream(disconnectMessage);
 
-        const canRetry = error !== undefined ? canRetryOnError(error) : true;
+        const canRetry = (error === undefined) || canRetryOnError(error);
```

Comment:
Not saying this is wrong, considering the code being logically equivalent. But I'm curious why we'd retry on an undefined error. It seems like this method might be used for a generic reconnect without an error, which is slightly against the name of the method. Nothing actionable right now but something to keep in mind in case of a larger refactoring.